api/src/main/java/org/openmrs/module/ModuleUtil.java

```diff
@@ -211,6 +212,35 @@ public static File insertModuleFile(InputStream inputStream, String filename) {
 		
 		return file;
 	}
+
+	/**
+	 * Checks if the current OpenMRS version is in an array of versions.
+	 * <p>
+	 * This method calls {@link ModuleUtil#matchRequiredVersions(String, String)} internally.
+	 * </p>
+	 *
+	 * @param versions the openmrs versions to be checked against the current openmrs version
+	 * @return true if the current openmrs version is in versions and false otherwise
```

Comment:
How phrasing this as "return true if the current openmrs version is in versions otherwise false"?